Jenkinsfile_CNP

```diff
@@ -13,7 +13,7 @@ def product = "sscs"
 def component = "bulk-scan"
 
 def secrets = [
-  'sscs-${env}'      : [
+  'sscs-bulk-scan-${env}'      : [
```

Comment:
is this right? How did it work before without `bulk-scan` appended to it?

Reply:
yeah, I asked myself the same question. I guess the answer is because either we use the same values for the secrets defined or because of the line setVaultName('sscs-bulk-scan') it was using the bulk vault anyway.